packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -308,7 +308,7 @@ export class DocumentDeltaConnection
 		return this.details.initialClients;
 	}
 
-	protected emitMessages(type: string, messages: IDocumentMessage[][]) {
+	protected emitMessages(type: string, messages: unknown[][]) {
 		// Although the implementation here disconnects the socket and does not reuse it, other subclasses
```

Comment:
This is not good for ops case. Is there something better?

Reply:
(IDocumentMessage | unknown)[][] ?

Reply:
I actually see Gary used `(IDocumentMessage | unknown)[][] | ISentSignalMessage[]` in his original commit -- I'll change to this for now

Reply:
`unknown` still seems like a problem. Isn't `Foo | unknown` just `unknown`?

Reply:
Yeah you're right it just becomes `unknown`. I think the solution is `IDocumentMessage[][] | ISentSignalMessage[]` and leave `content: IDocumentMessage` in submitSignal signature

Reply:
Yes that's right -- will insted use `(IDocumentMessage[] | ISentSignalMessage)[]`